### DISCUSSION TITLE
shairport-sync DACs on Raspberry Pi5

### DISCUSSION THREAD

**Discussion by petcol** [mikebrady/shairport-sync#1854] [Q&A] [answered]
I've just only noticed the Pi5 doesn't have a 3.5 mm Analogue Audio Output jack onboard...
Glad I didn't race out an buy one.
has any one got a solution other than a USB - DAC - 3.5mm adaptor I do have a couple floating but not sure if I want to go down that track.

**Comment by giggywithit** [comment, marked as answer]
dac hat
look around
theyre on amazon, all over

**Comment by microfx** [comment]
using pisound for years ...and it also noved from my Pi4 to my Pi5. Works perfectly. Probably one of the best quality DACs
